Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN npm run build
 #
 # Build backend
 #
-FROM golang:1.17-alpine as backend-builder
+FROM golang:1.19-alpine as backend-builder
```

Comment:
why not 1.20?

Reply:
Still waiting for mac 1.20 version, but for builder this wouldn't matter....